pkg/helpers/open/open.go

```diff
@@ -51,3 +53,14 @@ func FindLink(proj *project.Project, linkName string) (url string, err error) {
 
 	return
 }
+
+func PrintLinks(proj *project.Project) (err error) {
+	if len(proj.Manifest.Open) == 0 {
+		return fmt.Errorf("no links found in the project")
+	}
+	for title, url := range proj.Manifest.Open {
+		fmt.Println(color.Green(title), "\t", url)
+	}
+
+	return nil
+}
```

Comment:
This function is not really in the scope of the `open` helper. 
It's fine to keep it here for now, but the fuzzy search adds some more code related to links, I think we should create a `links` helper for that.

Reply:
I think you're right. Knowing how to print a link is not really the concern of the `open` helper.

Reply:
Incremental changes! Let's ship that and keep it in mind!